.env.example

```diff
@@ -22,11 +22,13 @@ REDIS_URL=redis://h:null@127.0.0.1:6379
 
 # AWS
 AWS_ACCESS_KEY_ID=
-AWS_REGION=us-east-1
+AWS_DEFAULT_REGION=us-east-1
 AWS_SECRET_ACCESS_KEY=
 AWS_S3_BUCKET=
 AWS_S3_REGION=us-east-1
 AWS_S3_URL=
+AWS_SQS_PREFIX=
+AWS_SQS_QUEUE=
```

Comment:
While Laravel keeps these unprefixed, I think we're better off prefixing them so all the `AWS_` vars group up nicely in the Heroku config settings interface and easier to see/edit.

Reply:
Are these two keys new?

Reply:
They exist as `SQS_PREFIX` and `SQS_QUEUE` in our Rogue config in Heroku. New in that I'm renaming them to add the `AWS_` prefix so all AWS related keys group up together in Heroku.

Reply:
Ah so just missing from the `.env.example` previously - love the renaming!

Reply:
Sorry I'm not sure where these are referenced in the code, but I don't see any updates and would have expected to see somewhere `SQS_PREFIX` being changed to `AWS_SQS_PREFIX` or did that already happen?

Reply:
They get used in the [`config/queue.php`](https://github.com/DoSomething/rogue/pull/1081/files#diff-baaf2a02f4c1142ceaa77e6baf4442d0R56-R57).

Reply:
Oh right - turns out I had already checked on that and marked as viewed 🤦‍♀️